modules/about-update-channels.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installation assemblies
+
+:_content-type: CONCEPT
+[id="about-update-channels_{context}"]
+= About update channels
+
+When you install an OADP Operator, you choose an _update channel_. This channel determines which upgrades to the OADP Operator and to Velero you receive. You can switch channels at any time.
+
+There are three update channels:
+
+* The *stable* channel contains the latest minor updates (y-stream updates) and patches (z-stream updates) of link:https://olm.operatorframework.io/docs/concepts/crds/clusterserviceversion/[`OADP ClusterServiceVersion`]. As each new release is published, the available `ClusterServiceVersion` of the OADP Operator will be appended with the latest available minor patch.
+
+* The *stable-1.0* channel contains `oadp.v1.0._z_`, the most recent OADP 1.0 `ClusterServiceVersion`.
+
+* The *stable-1.1* channel contains `oadp.v1.1._z_`, the most recent OADP 1.1 `ClusterServiceVersion`.
+
+*Which update channel is right for you?*
+
+* Choose the *stable* update channel to install the latest stable OADP version and to receive both minor updates and patches. If you choose this channel, you will receive all y-stream and all z-stream updates for version _x.y.z_.
+
+* Choose the *stable-1.0* update channel to install OADP 1.0 and to continue receiving patches for it. If you choose this channel, you will receive all z-stream patches for version 1.0._z_.
+
+* Choose the *stable-1.1* update channel to install OADP 1.1 and to continue receiving patches for it. If you choose this channel, you will receive all z-stream patches for version 1.1._z_.
+
+*When do you need to switch upgrade channels?*
+
+* If you have OADP 1._y_ installed, with _y_ greater than 0, and you want to receive patches only for that y-stream, you need to switch from the *stable* update channel to the *stable-1._y_* update channel. You will then receive all z-stream patches for version 1._y_._z_.
```

Comment:
please remove `, with _y_ greater than 0` and place it on line 33. Doesn't fit here as it applies also for 1.0.z releases